src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -104,7 +104,7 @@
       }
 
   type_mappings:
-    - input_type: int64
+    - input_type: datetime

```

Comment:
will this field always represent the input for the shared library as opposed to the input for the featurizer itself?

Reply:
The meaning of datetime changes based on context...

- From the shared library's perspective / C++, it is: std::chrono::system_clock::time_point
- From the C interface's perspective, it is: ISO 8601 string, posix time (int64)
- From the ML.Net interface's perspective, it is: System.DateTime, ISO 8601 string, and (potentially) posix time (int64)
- From a potential Python interface's perspective, it is: datetime.datetime, ISO 8601 string, and (potentially) posix time (int64)